templatefunc.go

```diff
@@ -496,6 +502,21 @@ func parseFormTag(fieldT reflect.StructField) (label, name, fType string, id str
 			label = tags[2]
 		}
 	}
+
+	return
```

Comment:
Looks like the diff is backwards lol, must be a github issue, I don't even see this locally, but look at the entire file around these lines, the end result should be good